src/OpenTelemetry/Trace/TracerProviderSdk.cs

```diff
@@ -30,8 +30,8 @@ internal class TracerProviderSdk : TracerProvider
         private readonly ActivityListener listener;
         private readonly Resource resource;
         private readonly Sampler sampler;
+        private readonly ActivitySourceAdapter adapter;
```

Comment:
Unrelated, VS was suggesting to make `ActivitySourceAdapter` readonly so I cleaned it up.